### PR TITLE
Mitschriften System-Entwurf 25.03.2021

### PR DESCRIPTION
**Pull request by RvNovae**

### CODE REVIEW

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
unchanged:
```



</details>

</details>

## Changed files
<details>
<summary>Changed 5 files: </summary>

- ABWL-LEDER/README.md
- ASM-GUENTHER/README.md
- DBS-DORENDORF/README.md
- PRO-KASCHE/README.md
- SWE-KASCHE/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- SWE-KASCHE/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

**Review by Steve2955 — CHANGES_REQUESTED** [review]
Siehe #3

**Review by Steve2955 — APPROVED** [review]

File: SWE-KASCHE/README.md [hunk 1]
```diff
@@ -3,7 +3,110 @@ Systemanalyse
...
+
+> User Story Format ("Als ... möchte ich ... um ... zu erreichen")
+>
+> Kommunikation
+>
+> AGILE: "How to slice a cake"
+
+### Kostenschätzungen
+
+- OO Sicht
```
Comment (ZeroPointMax) [inline SWE-KASCHE/README.md:45]:
```suggestion
- objektorientierte Sicht (OO-Sicht)
```

**Review by ZeroPointMax — APPROVED** [review]
👍

**Pull request merged**